.github/workflows/deploy-docs-on-netlify.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish docs on release
+
+on:
+  push:
+    paths:
+      - "docs/**"
+    tags:
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2
+        with:
+          version: 6
+
+      - name: Set node version to 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "pnpm"
+
+      - name: Install deps
+        run: pnpm install
+
+      - name: Build docs
+        run: pnpm run ci-docs
+
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          publish-dir: "docs/.vitepress/dist"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          production-deploy: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
This will also match for beta releases, no? That we wanted to avoid 
https://github.com/vitejs/vite/releases/tag/v2.9.0-beta.11

Reply:
yes，maybe a more specific regular expression ..

Reply:
I use `contains` which is defined as built-in function 

( docs:  https://docs.github.com/en/actions/learn-github-actions/expressions#contains )